cpg_workflows/metamist.py

```diff
@@ -83,7 +83,7 @@
 
 GET_PEDIGREE_QUERY = gql(
     """
-        query MyQuery($metamist_proj: String!){
+        query PedgireeQuery($metamist_proj: String!){
```

Comment:
PedigreeQuery!

Reply:
much prefer the sound of Pedgiree tbh, sounds like some sort of fancy cheese :D